frontend/src/components/QuestionCandidateFilter.vue

```diff
@@ -15,10 +15,23 @@ const emit = defineEmits(['update:modelValue']);
 
 const electionStore = useElectionStore();
 
-const uniqueCandidateIds = new Set<string>(
-  electionStore.calculator?.candidates.map((x) => x.id)
+const candidates = electionStore.calculator?.candidates;
+
+const uniqueCandidateIds = new Set<string>(candidates?.map((x) => x.id));
+const defaultSelectedCandidateIds = new Set<string>(
+  candidates
+    ?.filter((x) => {
+      if (
+        x.id === 'b0a9f173-d83d-49d4-8e8f-4fbddcfcbdac' ||
```

Comment:
```suggestion
        x.id === 'b0a9f173-d83d-49d4-8e8f-4fbddcfcbdac' ||
```

Ne, úplně jsme se nepochopili, měl by tam být flag `is_active`, podívej se, jak to je normálně ve výsledcích (ten checkbox _Zobrazit i nepostupující kandidáty_).